src/m365/tenant/commands/serviceannouncement/serviceannouncement-message-get.ts

```diff
@@ -0,0 +1,78 @@
+import { Logger } from '../../../../cli';
+import GraphCommand from '../../../base/GraphCommand';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+export interface Options extends GlobalOptions {
+  id: string;
+}
+
+class TenantServiceAnnouncementMessageGetCommand extends GraphCommand {
+  public get name(): string {
+    return commands.SERVICEANNOUNCEMENT_MESSAGE_GET;
+  }
+
+  public get description(): string {
+    return 'Retrieves a specified service update message for the tenant';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.id = (!(!args.options.appId)).toString();
```

Comment:
That was supposed to be `!args.options.id` - do you think message id could be added to telemetry or it's unnecessary?

Reply:
Only optional flags should be tracked. This option is required so you can leave it out

Reply:
I just realized that what I wanted was to track the value of `args.options.id`. The code I copied would only track whether it's set or not 🤦‍♂️

Reply:
With the telemetry we just want to track wich options are being used and not their values. Since the id is always used there is no benefit of tracking it

Reply:
Telemetry removed in 3fc651c